package.json

```diff
@@ -40,7 +40,8 @@
     "node-fetch": "^2.6.0",
     "qs": "^6.11.1",
     "reflect-metadata": "^0.1.13",
-    "url-parse": "^1.4.3" 
+    "url-parse": "^1.4.3",
+    "@stoplight/prism-cli": "5.6.0"
```

Comment:
This should not be a dependency for the node sdk. Its a dev dependency only. Should not be part of what we ship to production.

Reply:
Understood will make that change in the latest commit.